packages/color-area/test/color-area.test.ts

```diff
@@ -160,13 +161,16 @@ describe('ColorArea', () => {
 
         await elementUpdated(el);
 
-        expect(el.x).to.equal(0.6666666666666666);
+        expect(el.x).to.equal(0.67);
         expect(el.y).to.equal(0.25);
     });
     it('accepts "Arrow*" keypresses with alteration', async () => {
         const el = await fixture<ColorArea>(
             html`
-                <sp-color-area color="hsla(100, 50%, 50%, 1)"></sp-color-area>
+                <sp-color-area
+                    color="hsla(100, 50%, 50%, 1)"
+                    style="--spectrum-global-dimension-size-2400:192px"
```

Comment:
Can you check if we ended up needing this?

Reply:
Doesn't seem like it.